docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,95 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
+
+1. Copy your AWS Long Term Access Key.
```

Comment:
Where do they use the AWS keys? I don't think they need them for any of these steps.

Reply:
@laurenfrederick This is for onboarding. And I took the steps from NSIDC and PO.DAAC resources for their onboarding setup for operators.

Reply:
The keys are not used in this document and they are not needed for any steps below, so I think this would be confusing for users. For dashboard access, keys are not needed.

Reply:
Okay.

Reply:
@laurenfrederick Done.